submissions/benjaminKamena/Dom/signUp.js

```diff
@@ -0,0 +1,58 @@
+// sign-up form validation
+
```

Comment:
This file exist already. Remove it from this pull request. I think you should sync your branch so you can get the file from the previous pull request @BenjaminKamena